Source/Atomic/UI/UI.cpp

```diff
@@ -150,7 +150,7 @@ void UI::HandleExitRequested(StringHash eventType, VariantMap& eventData)
 
 void UI::Shutdown()
 {
-    SetInputDisabled(true);
+    //SetInputDisabled(true);
```

Comment:
it's really unnecessary to disable UI input after requesting exit. Should I just remove it?